src/app/pages/landingpage/landingpage.component.html

```diff
@@ -36,9 +36,63 @@
             <button class="bg-transparend text-teal-500 py-5 px-5 font-bold">Learn more ></button>
           </div>
         </div>
+        <div class="w-1/2" style="min-height: 600px;">
+          <div class="animated_slot_machine">
+            <div class="outer_border" ></div>
+            <div class="top_wide_border_segment" ></div>
+            <div class="top_bottom_border_segment" ></div>
+            <div style="font-family: 'Roboto Mono', sans-serif;" class="font-light text-lg price_div" >PRICE/HOURS</div>
```

Comment:
let's add `roboto mono` to tailwind? https://tailwindcss.com/docs/font-family#customizing-your-theme